lib/matplotlib/axes/_axes.py

```diff
@@ -1616,36 +1616,44 @@ def plot_date(self, x, y, fmt='o', tz=None, xdate=True, ydate=False,
     @docstring.dedent_interpd
     def loglog(self, *args, **kwargs):
         """
-        Make a plot with log scaling on both the *x* and *y* axis.
+        Make a plot with log scaling on both the x and y axis.
 
-        :func:`~matplotlib.pyplot.loglog` supports all the keyword
-        arguments of :func:`~matplotlib.pyplot.plot` and
-        :meth:`matplotlib.axes.Axes.set_xscale` /
-        :meth:`matplotlib.axes.Axes.set_yscale`.
+        Call signatures::
+
+            loglog([x], y, [fmt], data=None, **kwargs)
+            loglog([x], y, [fmt], [x2], y2, [fmt2], ..., **kwargs)
+
+        This is just a thin wrapper around `.plot` which additionally changes
+        both the x-axis and the y-axis to log scaling. All of the concepts and
+        parameters of plot can be used here as well.
+
+        The additional parameters *basex/y*, *subsx/y* and *nonposx/y* control
+        the x/y-axis properties. They are just forwarded to `.Axes.set_xscale`
+        and `.Axes.set_yscale`.
 
         Parameters
         ----------
-        basex, basey : scalar
+        basex, basey : scalar, optional, default 10
```

Comment:
"must be >1" is actually not true (although this could arguably be discussed in a separate issue)

Reply:
Should I write something else? The `semilogx/y` use the weaker phrase "The value should be larger than 1.".

I tested that I can generate plots with 0 < base < 1. No idea if there's a use-case for that. Negative values are accepted, but generate an error when drawing the plot (problably when updating the axes). Invalid values should be rejected by `set_xscale` (or probably a level deeper, which I didn't follow through). Should I open an issue on that?

Reply:
I think there isn't any real reason to reject 0<base<1, but given that base==1 won't work (of course), it may be simpler to indeed just spec base>1, or not say anything about it.
I think that's the job of the scale classes anyways.

Reply:
Ok, I'll remove it. It's not particularly helpful, to say something about the allowed values. People will know what base they want. And the scales should complain if they cannot handle it.

Reply:
Note: Removed. This is not reflected here, because it's on the next line.